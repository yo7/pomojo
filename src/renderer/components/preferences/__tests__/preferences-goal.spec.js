import Vue from 'vue'
import Vuex from 'vuex'
import {shallow} from 'vue-test-utils'
import {createRenderer} from 'vue-server-renderer'
import PreferencesGoal from '../preferences-goal.vue'

Vue.use(Vuex)

describe('PreferencesGoal', () => {
  const actions = {
    'preferences/updateGoal': jest.fn()
  }
  const store = new Vuex.Store({actions})
  const propsData = {goal: 8}

  it('renders correctly', () => {
    const wrapper = shallow(PreferencesGoal, {store, propsData})
    const renderer = createRenderer()
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) {
        throw new Error()
      }
      expect(str).toMatchSnapshot()
    })
  })
})
