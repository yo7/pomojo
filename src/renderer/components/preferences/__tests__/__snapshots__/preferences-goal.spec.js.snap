// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreferencesGoal renders correctly 1`] = `
<div class="preferences-goal">
    <!---->
</div>
`;
